docs/Concepts/Weak-Subjectivity.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Describe weak subjectivity
+---
+
+# Weak subjectivity
+
+The weak subjectivity period refers to how far behind the chain head a node can be before knowing
+that 1/3 of validators have exited.
```

Comment:
Feels slightly ambiguous.

```suggestion
The weak subjectivity period refers to how far behind the chain head a node can be before 1/3 of validators may have exited without the node knowing.
```

or: may have have existed since the node was last in sync.